core/indexer/elasticClientCommon.go

```diff
@@ -77,6 +77,10 @@ func elasticDefaultErrorResponseHandler(res *esapi.Response) error {
 		if errIsAlreadyExists(responseBody) {
 			return nil
 		}
+		if errAliasAlreadyExits(responseBody) {
+			log.Debug("alias already exits", "response", fmt.Sprintf("%v", responseBody))
```

Comment:
exists*

Reply:
done.

Reply:
you can drop `fmt.Sprintf("%v", ` and pass directly the response body to the log.Debug function call as this internally does a `sprintf("%v")`